posthog-react-native/src/posthog-rn.ts

```diff
@@ -451,4 +452,29 @@ export class PostHog extends PostHogCore {
     this.setPersistedProperty(PostHogPersistedProperty.InstalledAppBuild, appBuild)
     this.setPersistedProperty(PostHogPersistedProperty.InstalledAppVersion, appVersion)
   }
+
+  /**
+   * @todo Where should this go, and should the result be cached?
+   * I can't find a public method in PostHog which does an authenticated fetch,
+   * so for now I've included the fetch here so it can access the api key.
+   */
```

Comment:
https://github.com/PostHog/posthog-js/blob/8d70a9ba4383b453ade94b71d7f45df4af2c892c/src/posthog-core.ts#L1294C25-L1321
i think those methods can be in the core package, [here](https://github.com/PostHog/posthog-js-lite/blob/main/posthog-core/src/index.ts) and reuse the fetch infrastructure from there, example [here](https://github.com/PostHog/posthog-js-lite/blob/a36099a183f3a1dbdd7a68a8d5c98d39e93019ae/posthog-core/src/index.ts#L294-L323).